ui/package.json

```diff
@@ -126,7 +126,7 @@
     "ember-a11y-testing": "^5.2.1",
     "ember-auto-import": "2.6.3",
     "ember-basic-dropdown": "6.0.1",
-    "ember-cli": "~4.12.1",
+    "ember-cli": "~4.12.2",
```

Comment:
We don't need to bump ember-cli right?

Reply:
No, but it seemed like it couldn't hurt as the patch just included a small bug fix? https://github.com/ember-cli/ember-cli/releases/tag/v4.12.2